packages/sdk/test/client.ts

```diff
@@ -0,0 +1,7 @@
+import { VoucherifyServerSide } from '../../sdk/src'
+import 'dotenv/config'
+export const voucherifyClient = VoucherifyServerSide({
```

Comment:
Make more specific error message when someone wants to run a test but does not configure required environment variables.


e.g.

```ts

if(!process.env.APPLICATION_ID){
  // throw new Error(.....)
}
```